docs/pages/contributing/documentation/reference.mdx

```diff
@@ -101,6 +117,92 @@ Partials will only be included in these two cases:
 
 These will be inserted as-is, even in the case of `.mdx` files.
 
+### Partial parameters
+
+A partial can define parameters, which allow multiple docs pages to include the
+same partial but with slightly different content.
+
+Inside a partial, a parameter uses the `{{ }}` syntax. For example, the partial
+`mypartial.mdx` below uses the `{{ action }}` parameter.
+
+```markdown
+This is an example of a partial.
+
+A partial makes it possible to {{ action }}.
+```
+
+To provide a value for a partial, the page that includes it can use the
+following syntax:
+
+```text
+(\!mypartial.mdx param1="val" param2="val"\!)
+```
+
+For example, a page could supply a value to the `action` parameter in the
+`mypartial.mdx` partial above using this expression:
+
+```text
+(\!mypartial.mdx action="reuse content in docs pages."\!)
+```
+
+This partial would render as:
+
+```markdown
+This is an example of a partial.
+
+A partial makes it possible to reuse content in docs pages.
+```
+
+When including a partial, parameter definitions must be separated by single
+spaces. Values must be enclosed in double quotes. Parameter names and values
+must be separated by a `=` character, and the `=` character must not be
+surrounded by spaces. 
+
+If a partial defines parameters but a page does not define them when including
+the partial, the docs engine will leave the corresponding `{{ }}` expressions
+untouched. The exception to this is when the partial defines default parameter
```

Comment:
Can you clarify what you mean by "untouched"? Does that mean it will render nothing, or that it will render, in your example above, the literal string "{{ action }}"?